### ISSUE TITLE
LibreELEC 9.x support?

### ISSUE THREAD

**Issue by krisrok** [veldenb/plugin.program.moonlight-qt#4]
I guess I already know the answer but why not ask ;)

Is it possible to backport this addon to 9.x?

**Comment by veldenb** [comment]
Hi,

The biggest difference between 9.0 and 10.0 is the upgrade from python 2 to 3. So this would require adjustments to the plug-in in a way that a separate plug-in is needed and I'll have to maintain 2 plug-ins instead of one.
Eventually everyone will probably upgrade to 10 so investing time in a backport is in a way a waste of time.

I might do a backport if I have enough spare time and there is enough interest but no promises.
It will also depend on the ability of LE 9.0 to run Moonlight and Docker, I haven't tried that yet.

If someone wants to try, feel free to create a PR.


TL;DR: Maybe :)

**Comment by krisrok** [comment]
Hey, I understand, no problem.

Regarding the 9.0 + docker issue: With https://github.com/clarkemw/moonlight-embedded-launcher it works fine already, at least as far as i can tell. You just have to choose the second-latest docker image (i think *26 instead of *27).

**Comment by veldenb** [comment]
Closing for now
